src/features/animation/embed-view.tsx

```
@@ -16,174 +16,255 @@ import { trackAnimationEvent } from "@/features/animation/analytics";
+        crossOrigin="anonymous"
+      />
+      <link
+        rel="stylesheet"
+        href={fonts[fontFamily].src}
+        crossOrigin="anonymous"
+      />
+
+      <div className="relative w-full h-screen overflow-hidden flex items-center justify-center bg-background">
+        {/* Aggressive letter-spacing reset for embed view */}
+        <style
+          dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)